bundle/README.md

```diff
@@ -1,16 +1,5 @@
 # CRI-O static build bundle
```

Comment:
I know this is pre-existing, and probably known by 99% of the people reading this, but should we add something like:

To install the bundle, run the following from a development sandbox:

Reply:
Fixed :+1: